packages/artillery/lib/cmds/run.js

```diff
@@ -529,6 +536,20 @@ async function sendTelemetry(script, flags, extraProps) {
   }
 }
 
+function checkDirExists(output) {
+  if (!output) {
```

Comment:
I don't think we need this check here because this function will be called only if `flags.output` is truthy already.